app.rb

```diff
@@ -1,6 +1,25 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative 'lib/peep'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/peeps' do
+    @peeps_return = Peep.all
+    erb :peeps
+  end
+
+  get '/peeps/post' do
+    erb :post
+  end
+
+  post '/peeps/new' do
+    Peep.post(params[:message])
+    redirect '/peeps'
+  end
```

Comment:
Your routs are quite close to the standard RESTful routing conventions, but not quite! If you'd like to use more standard names for you routes, check out this blog post on RESTful routing: https://medium.com/@shubhangirajagrawal/the-7-restful-routes-a8e84201f206